src/pages/on-chain-governance/_components/ProposalDetail.tsx

```diff
@@ -0,0 +1,189 @@
+import BigNumber from "bignumber.js";
+import classNames from "classnames";
+import { AiFillGithub } from "react-icons/ai";
+import { NumericFormat } from "react-number-format";
+import { format, fromUnixTime } from "date-fns";
+import {
+  ProposalInfo,
+  ProposalType,
+  ProposalStatus,
+} from "@defichain/jellyfish-api-core/dist/category/governance";
+import { Link } from "@components/commons/link/Link";
+import { ProposalDisplayName } from "./ProposalCard";
+
+export function ProposalDetail({
+  proposal,
+  proposalEndMedianTime,
+  proposalCreationMedianTime,
+}: {
+  proposal: ProposalInfo;
+  proposalEndMedianTime: number;
+  proposalCreationMedianTime: number;
+}) {
+  const dateFormat = "MM/dd/yyyy";
+  const endDate = format(fromUnixTime(proposalEndMedianTime), dateFormat);
+  const startDate = format(
+    fromUnixTime(proposalCreationMedianTime),
+    dateFormat
+  );
+
+  const blockPage =
+    proposal.status === ProposalStatus.VOTING
+      ? "/blocks"
+      : `/blocks/${proposal.cycleEndHeight}`;
+
+  return (
+    <div className="md:border md:p-6 border-gray-200 rounded-lg">
+      <div className="flex mb-2">
+        <div className="p-2 bg-gray-100 dark:bg-gray-800 rounded">
+          <span className="text-sm font-medium text-gray-900 dark:text-gray-100 tracking-[0.015em]">
+            {ProposalDisplayName[proposal.type]}
```

Comment:
can you check the styling of this element, it looks slightly longer

Reply:
double checked on this padding and text size is the same

Reply:
<img width="69" alt="image" src="https://user-images.githubusercontent.com/40191153/210529919-4e3f69b8-2be2-4b64-b347-8e7ebf971ea9.png">
<img width="72" alt="image" src="https://user-images.githubusercontent.com/40191153/210530063-0e9d19d6-dd30-476d-bcdf-6e7348361b41.png">
both taken at 100%, one from figma one from browser, they just dont like the same

Reply:
updated to use div instead of span